sdk/include/psdk/d3dkmdt.h

```diff
@@ -600,12 +554,12 @@ typedef struct _D3DKMDT_VIDEO_SIGNAL_INFO
         struct
         {
             // Scan line ordering (e.g. progressive, interlaced).
-            D3DDDI_VIDEO_SIGNAL_SCANLINE_ORDERING ScanLineOrdering : 3;
-
+            D3DDDI_VIDEO_SIGNAL_SCANLINE_ORDERING ScanLineOrdering;
+            UINT Reserved                       : 2;
```

Comment:
This doesn't seem to make sense. If you remove the bit-width from ScanLineOrdering, it will be a full field, and those 2 bits of `Reserved` are pointless waste of space. The question is: what is wrong with `ScanLineOrdering : 3;`?

Reply:
Actually this doesn't even get build anywhere so I am not even sure why i did this.

Reply:
error: 'ScanLineOrdering' is narrower than values of its type [-Werror]
             D3DDDI_VIDEO_SIGNAL_SCANLINE_ORDERING ScanLineOrdering : 3;
             
 Oh yeah, it's an enum.
 
 here ill make it a UINT type and add a comment so it doesn't break.